scripts/sql/schema.sql

```diff
@@ -1,101 +1,94 @@
 -- schema for reporting db
-create table url_metadata (
-    id int(10) not null auto_increment,
-    query varchar(1024),
-    fragment varchar(320),
-
+create table users (
+    id serial not null unique,
+    email varchar(2083) not null unique,
+    count integer not null,
     primary key (id)
 );
 
-create table hostnames (
-    id int(10) not null auto_increment,
+
+create table urls (
+    id serial not null unique,
     host varchar(2083) not null,
+    path varchar(2083) not null,
```

Comment:
why 2083 specifically?

Reply:
I think I looked up the theoretical max of a hostname and it was around 2083.